setup.cfg

```diff
@@ -25,7 +25,7 @@ install_requires =
     nipype ~= 1.5.1
     niworkflows >= 1.3.2
```

Comment:
```suggestion
    niworkflows >= 1.4.0rc5
```

Although, this is going to bring a whole host of API changes, probably.

Reply:
Well, better to discover them now...